tools/packaging/qemu/patches/5.1.x/0014-gitmodules-use-GitLab-repos-instead-of-qemu.org.patch

```diff
@@ -0,0 +1,111 @@
+From 104e56711f131d80de82caed8759947509576e3b Mon Sep 17 00:00:00 2001
```

Comment:
> It wouldn't hurt to mention the QEMU upstream commit this was backported from, but anyway the patch is fine.

@gkurz isn't this a mention

Reply:
Hmm I don't know where this hex blob is coming from but it isn't a valid SHA1 in the upstream QEMU git repo.

Original commit is :

https://gitlab.com/qemu-project/qemu/-/commit/9911ca0d1bca846f159ebdb48b9d8a784c959589

Reply:
Ah yes. Of course it's different because the patch is different.